doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -149,11 +150,14 @@ Please refer to the  flow/sequence diagrams which covers the following required
 # Feature enablement
   This feature (optics Interface Link bring-up sequence) would be enabled on per platform basis.
   There could be cases where vendor(s)/platform(s) may take time to shift from existing codebase to the model (work-flows) described in this document.
-  In order to avoid any breakage and ensure gradual migration of different platforms/vendors to this model, there would be new field (flag) in xcvrd to enable/disable this feature. 
-  When xcvrd spawns on LC/board, it would invoke platform plugin to check with the platform (hwsku) whether this feature is yet supported on underlying platform (board/LC) or not
+  In order to avoid any breakage and ensure gradual migration of different platforms/vendors to this model, will add this new workflow to enable/disable this feature:
+  
+  In order to enable this feature, the platform would set ‘skip_xcvrd_cmis_mgr’ to ‘false’ in their respective pmon_daemon_control.json as part of platform bootstrap. When xcvrd would spawn on that hwsku (LC/board), it would parse ‘skip_xcvrd_cmis_mgr’ and if found 'false', it would launch CMIS task manager. This implies enabling this feature. 
+Else, if ‘skip_xcvrd_cmis_mgr’ is set/found 'true' by xcvrd, it would skip launching CMIS task manager and this feature would remain disabled.
   
```

Comment:
there is a 3rd case what if a platform/vendor doesn't specify 'skip_xcvrd_cmis_mgr'. What should be the default?

Reply:
> there is a 3rd case what if a platform/vendor doesn't specify 'skip_xcvrd_cmis_mgr'. What should be the default?

In that case, xcvrd to assume/consider  'skip_xcvrd_cmis_mgr' as 'true' i.e. do not launch CMIS task manager.

Reply:
@shyam77git can we keep the default (i.e the platform if it doesn't specify 'skip_xcvrd_cmis_mgr') then enable cmis mgr. ? i.e default is 'False'

Reply:
> @shyam77git can we keep the default (i.e the platform if it doesn't specify 'skip_xcvrd_cmis_mgr') to Enable cmis mgr. ? i.e default is 'False'

@prgeor - won't that become an issue from system standpoint (coming-up or reliability)? 'cause if platform is not clarifying (mentioning) this flag and SONiC/xcvrd takes it as 'False' i.e. initiate CMIS spec/mgr workflow, it would break IMO - right? platform may not come up or go down!

Reply:
No. cmis mgr is already in use for quite sometime now i.e if xcvrd finds QSFP-DD, then it will initialize the module (provided, the platform is using sfp-refactor). The disable knob is for future if any platform doesn't want to use CMIS mgr. Can you update the doc with this default behavior.